### ISSUE TITLE
Support undo/redo history across restarts

### ISSUE THREAD

**Issue by pablolagreca** [casouri/vundo#97]
It would be great to keep undo/redo history after restarts. It has a cost, maybe it can be limited by time and or size of history.

**Comment by casouri** [comment]
I believe you can archive that with https://melpa.org/#/undo-fu-session

**Comment by jdtsmith** [comment]
Yes, I use `undo-fu-session`.  One issue with it is that saving undo history tends to build up very long vundo lines of >5-20k, which emacs still doesn't like.

We fixed a bug (see #68) that speeds up draw 10x, but just having such long lines displayed in any buffer slows emacs down perceptibly (including vundo navigation).  Long term, it would be nice to draw "only what's needed" in the buffer to avoid this.  I'll open an issue.
